react-common/styles/controls/Checkbox.less

```diff
@@ -1,6 +1,6 @@
 .common-checkbox {
     position: relative;
-    padding: 0.25rem;
+    // padding: 0.25rem;
```

Comment:
Oh, shoot. I just forgot to remove that comment. I'll put it back in. From what I saw, it didn't make a difference, but I am not sure if it will impact other checkboxes.